### ISSUE TITLE
kepub page numbers doubled on some conversions

### ISSUE THREAD

**Issue by Second-Victor** [pgaskin/kepubify#101]
some epubs when converted to kepub have double the page numbers, but the first half of the pages are ignored by my Kobo Clara Color, the device jumps from page 1 to half way through the total.

The pages don't seem to be there unless I use the slider at the bottom of the screen and go back into the part that was jumped by the kobo. The pages are not blank and it is as if the book has been doubled even thought the file size is not

**Comment by pgaskin** [comment]
Is the TOC in the same order as the spine?

**Comment by Second-Victor** [comment]
yes I have checked the TOC order, I managed to fix the issue if I manually save the TOC in calibre after renaming all the html files.

I also asked other users on reddit
https://www.reddit.com/r/kobo/comments/1eulj05/kepub_files_with_double_page_count/

and found this on mobile read
https://www.mobileread.com/forums/showthread.php?t=361011

**Comment by pgaskin** [comment]
I will add a workaround for this in the version of kepubify.

**Comment by Huguesmmm** [comment]
### I have the same problem. I don't really care about the page's number, but here's something weird :
- When you use table of content and choose a chapter, it takes you to a "double" of the start of the chapter. 
- If you check on the slider, you noticed it's a part that is "jumped" or skipped when you just open the book and read. When you read a .kepub on a kobo that was converted, it always start at a certain number (ex. 500, 2000, etc.) but never 0.
### And here's the frustrating bit : 
- When you finish the chapter you chose in the table of content, it takes you back to you current reading position. So you from page 628 to 3318.

**Comment by MathisP75** [comment]
This is still a problem in 2026
